src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -371,6 +362,15 @@ private void walk(boolean isDefault) {
           visitEnumType((EnumType) type);
           return;
         }
+        // Closure keeps type NONE for the symbol which became the type alias.
+        // However, the aliased type is present in the registry under the same name.
+        if (type.isNoType()) {
+          JSType registryType = typeRegistry.getType(symbol.getName());
+          if (registryType != null) {
+            visitTypeAlias(registryType, getUnqualifiedName());
+            return;
+          }
```

Comment:
I am not that convinced NoType only appears with @typedef, so just rewording the comment for now.